README.md

```diff
@@ -20,7 +20,8 @@ Fork this repo; edit config.json and modify the first lines with your settings:
 	"poolState": "poollogs.json",                               // Where to save pool state
 	"paymentsFile": "payments.sh",                              // Where to save payments commands
 	"includeSelfStake": false,                                  // True if we want to include selfstake in distribution calculations
-	"multiSignature": false										// True if you're using a multisig account
+	"multiSignature": false,				    // True if you're using a multisig account
+	"fromAddress": ""					    // Address from where to make the payments. Unchanged if the payments will be made from the delegate's address
```

Comment:
// Address from where to make the payments. Remove if the payments will be made from the delegate's address